test/functional/Valhalla/src/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -182,18 +257,19 @@ static public void testCreatePoint2DComplex() throws Throwable {
 	 * }
 	 * 
 	 */
-	@Test(priority=2)
+	@Test(priority = 2)
 	static public void testCreateLine2D() throws Throwable {
-		String fields[] = {"st:LPoint2D;:value", "en:LPoint2D;:value"};
+		String fields[] = { "st:LPoint2D;:value", "en:LPoint2D;:value" };
 		line2DClass = ValueTypeGenerator.generateValueClass("Line2D", fields);
-		
-		makeLine2D = lookup.findStatic(line2DClass, "makeValue", MethodType.methodType(line2DClass, point2DClass, point2DClass));
-		
-		MethodHandle getSt = generateGetter(line2DClass, "st", point2DClass);
```

Comment:
the `getSt` and `getEn` can remain local since they are only used here

Reply:
they have to be `static`, since they will be used in helper functions.

Reply:
They dont need to be in the helper functions, they are only needed here